src/main/java/duke/exception/InvalidCommandException.java

```diff
@@ -0,0 +1,15 @@
+package duke.exception;
+
+/**
+ * Indicates that a user command is invalid
+ */
+public class InvalidCommandException extends Exception {
+
+    /**
+     * Constructor with a message
```

Comment:
Following checkstyles, there should be a fullstop after the first line of a JavaDoc.